cypress/e2e/api.cy.js

```diff
@@ -0,0 +1,12 @@
+describe('EngageSphere - API', () => {
```

Comment:
```suggestion
describe('EngageSphere', () => {
```

Visto que o nome do arquivo se chama `api.cy.js`, será que realmente precisamos desta _string_ na descrição do teste?

Fica a provocação para pensar no assunto.

Reply:
Bom ponto, realmente fica um pouco redundante, vou fazer a alteração aqui.